doc/source/notebooks/basics/regression.pct.py

```diff
@@ -179,6 +179,9 @@
 _ = plt.xlim(-0.1, 1.1)
 
 
+# %% [markdown]
+# Moreover, you can get the mean and variance data individually for example as `m.predict_f(xx).mean` instead of `m.predict_f(xx)[0]`.
```

Comment:
```suggestion
# Moreover, you can get the mean and variance predictions individually, using for `m.predict_f(xx).mean` instead of `m.predict_f(xx)[0]` and `m.predict_f(xx).variance` instead of `m.predict_f(xx)[1]`.
```
(note- if we rename the variance field we'll have to update this)